package.json

```diff
@@ -33,7 +33,6 @@
     "express-ws": "2.0.0-rc.1",
     "fs-promise": "0.5.0",
     "immutable": "3.7.6",
-    "lodash": "4.13.1",
```

Comment:
See https://github.com/mozilla/tofino/issues/647 
I think node_modules should be clobbered.